packages/core/src/utilities.ts

```diff
@@ -63,3 +63,16 @@ export function applyStyles(
 
   return flattenedStyles;
 }
+
+export function getRealValue(value: any) {
```

Comment:
```suggestion
export function getRealValue(value: string | number) {
```

i think ideally you want a string or number right?

Reply:
@peterpme done.